test/helpers/utils.go

```diff
@@ -140,6 +140,33 @@ func GetAppPods(apps []string, namespace string, kubectl *Kubectl, appFmt string
 	return appPods
 }
 
+// WaitCiliumEndpointReady gets cilium pod for a ginkgo node (k8s1/k8s2)
+// and waits for all the endpoints of that node to ve ready.
+func WaitCiliumEndpointReady(podFilter string, kubectl *Kubectl, k8sNode string) (string, EndpointMap) {
```

Comment:
Can you change this function so it has kubectl as its receiver? E.g., `func (kub *Kubectl) WaitCiliumEndpointReady`

Reply:
done